FinniversKit/Sources/DNA/Color/UIColor+FinniversKit.swift

```diff
@@ -7,17 +7,19 @@ import UIKit
 // MARK: - Semantic colors, dark mode compatible
 @objc extension UIColor {
     public class var accentPea: UIColor { Config.colorProvider.accentPea }
+    public class var accentPrimaryBlue: UIColor { Config.colorProvider.accentPrimaryBlue }
     public class var accentSecondaryBlue: UIColor { Config.colorProvider.accentSecondaryBlue }
     public class var accentToothpaste: UIColor { Config.colorProvider.accentToothpaste }
     public class var bgAlert: UIColor { Config.colorProvider.bgAlert }
     public class var bgBottomSheet: UIColor { Config.colorProvider.bgBottomSheet }
     public class var bgCritical: UIColor { Config.colorProvider.bgCritical }
+    public class var bgSuccess: UIColor { Config.colorProvider.bgSuccess }
     public class var bgInfo: UIColor { Config.colorProvider.bgInfo }
+    public class var bgInfoHeader: UIColor { Config.colorProvider.bgInfoHeader }
     public class var bgPrimary: UIColor { Config.colorProvider.bgPrimary }
-    public class var bgQuaternary: UIColor { Config.colorProvider.bgQuaternary }
     public class var bgSecondary: UIColor { Config.colorProvider.bgSecondary }
-    public class var bgSuccess: UIColor { Config.colorProvider.bgSuccess }
     public class var bgTertiary: UIColor { Config.colorProvider.bgTertiary }
+    public class var bgQuaternary: UIColor { Config.colorProvider.bgQuaternary }
```

Comment:
These were ordered alphabetically so that we don't have to maintain grouping across UIColor, CGColor and Color. 😄